src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb

```diff
@@ -13604,6 +13604,7 @@ BC36716: Visual Basic 14.0 does not support tuples.
                                       ~~~~~~
 </errors>)
 
+            Assert.Null(comp.GetRequiredLanguageVersion(comp.GetDiagnostics()(0)))
```

Comment:
The VB diagnostics for this scenario look like this:
```
BC36716: Visual Basic 14.0 does not support tuples.
        Dim x As (Integer, Integer) = (1, 1)
                 ~~~~~~~~~~~~~~~~~~
```
Note that the required version is not included. This could be fixed, but I'd rather avoid growing this PR further.